kiele

```diff
@@ -181,16 +198,17 @@ cmode='Lbl'$mode'{}()'
 cschedule='Lbl'$schedule'{}()'
 cenablecoverage='\dv{SortBool{}}("'$enablecoverage'")'
 
-if [[ "$run_command" != 'vm' ]]; then
+if [[ "$run_command" != 'vm' && "$run_command" != "generate-report" ]]; then
```

Comment:
```suggestion
if [[ "$run_command" != 'vm' ]]; then
```
With this `run_file` is already made for you so you don't have to declare `report_json`. It looks like you would need to handle `-` as a special case, just reassigning `run_file` to `/dev/stdin` should work.